Gruntfile.js

```diff
@@ -9,147 +9,9 @@
-                dest: 'build/dist/',
-                options: {
-                    // Remove 'aws-sdk' module
-                    process: function (content) {
-                        return content.replace(/\s*(var)?\s*aws\s*=\s*require\s*\(\s*'aws\-sdk'\s*\)\s*;/i, '');
-                    }
-                }
-            }
-        },
-
-        mochaTest: {
-            'server-side': mochaTestOptions('xunit', 'build/reports/server-side-tests.xml'),
-            'html-cov': mochaTestOptions('html-cov', 'build/reports/server-side-coverage.html'),
-            'cobertura': mochaTestOptions('mocha-cobertura-reporter', 'build/reports/server-side-coverage.xml'),
-            'travis-cov': {
-                options: {
-                    reporter: 'travis-cov',
-                    require: bootStrap
-                },
-                src: testSource
-            }
-        },
-
-        browserify: {
-            test: {
-                files: {
-                    'build/test/browser/ama-sdk.js': 'build/dist/lib/ama.js',
-                    'build/test/browser/specs.js': ['build/test/**/*.coffee']
-                },
-                options: {
-                    transform: ['coffeeify'],
-                    browserifyOptions: {
-                        extensions: ['.coffee']
-                    }
-                }
-            },
-            dist: {
-                files: {
-                    'dist/aws-sdk-mobile-analytics.js': 'build/dist/lib/ama.js'
-                }
-            }
-        },
-
-        uglify: {
-            test: {
-                files: {
-                    'build/test/browser/ama-sdk.min.js': 'build/test/browser/ama-sdk.js',
-                }
-            },
-            dist: {
-                files: {
-                    'dist/aws-sdk-mobile-analytics.min.js': 'dist/aws-sdk-mobile-analytics.js'
-                }
-            }
-        },
-
-        mocha: {
-            'client-side': {
-                src: ['build/test/browser/**/*.html'],
-                dest: 'build/reports/client-side-tests.xml',
-                options: {
-                    logErrors: true,
-                    reporter: 'XUnit',
-                    run: true,
-                    timeout: 10000,
-                    log: true
-                }
-            }
-        },
-
-        clean: {
-            build: {
-                src: ['build/**']
-            }
-        }
-    });
-
-    grunt.loadNpmTasks('grunt-contrib-jshint');
-    grunt.loadNpmTasks('grunt-contrib-copy');
-    grunt.loadNpmTasks('grunt-contrib-uglify');
-    grunt.loadNpmTasks('grunt-contrib-clean');
-
-    grunt.loadNpmTasks('grunt-blanket');
-    grunt.loadNpmTasks('grunt-mocha-test');
-
-    grunt.loadNpmTasks('grunt-browserify');
-    grunt.loadNpmTasks('grunt-mocha');
+    require('load-grunt-tasks')(grunt);
+    grunt.loadTasks('grunttasks');
 
     grunt.registerTask('default', ['npm-install']);
```

Comment:
What does this actually do? I do not see why this is needed, as there is no task named `npm-install`. Can this be removed @cheruvian ?